test/EndToEnd/GivenSelfContainedAppsRollForward.cs

```diff
@@ -13,13 +12,13 @@
 
 namespace EndToEnd
 {
-    public class GivenSelfContainedAppsRollForward : TestBase
+    public partial class GivenSelfContainedAppsRollForward : TestBase
     {
 
-        [Theory(Skip = "Runtime 1.1 support for openSUSE and Fedora 27 needed")]
```

Comment:
@dsplaisted new Linux lags are failing. At the same time, it is failing on assertion `floatedVersion` is 2.0.6 instead of 2.0.7. This is strange, it means 2.0.7 is not in the feed. Do you know why? It should be in myget at least

Reply:
@johnbeisner is the cause of https://github.com/dotnet/cli/pull/9202 fix? Can I reenable it?

Reply:
@dsplaisted that caught an actual bug

https://github.com/dotnet/cli/blob/508f902ed0d49891400a806e9b9785e5434d9789/build/MSBuildExtensions.targets#L174 will write out the following to disk. Which equal to no op. Not good...

```xml
    <LatestPatchVersionForNetCore1_0 Condition="'1.0.11' == ''">1.0.11</LatestPatchVersionForNetCore1_0>
    <LatestPatchVersionForNetCore1_1 Condition="'1.1.8' == ''">1.1.8</LatestPatchVersionForNetCore1_1>
    <LatestPatchVersionForNetCore2_0 Condition="'2.0.7' == ''">2.0.7</LatestPatchVersionForNetCore2_0>
```

Reply:
Nice catch.  Looks like the `$` needs to be escaped in MSBuildExtensions.targets.

Reply:
that is the end result now. Looks alright
```xml
<!--
***********************************************************************************************
Microsoft.NETCoreSdk.BundledVersions.props

WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
          created a backup copy.  Incorrect changes to this file will make it
          impossible to load or build your projects from the command-line or the IDE.

Copyright (c) .NET Foundation. All rights reserved.
***********************************************************************************************
-->
<Project>
  <PropertyGroup>
    <BundledNETCoreAppTargetFrameworkVersion>2.1</BundledNETCoreAppTargetFrameworkVersion>
    <BundledNETCoreAppPackageVersion>2.1.1-servicing-26605-02</BundledNETCoreAppPackageVersion>
    <BundledNETStandardTargetFrameworkVersion>2.0</BundledNETStandardTargetFrameworkVersion>
    <BundledNETStandardPackageVersion>2.0.3</BundledNETStandardPackageVersion>
    <BundledNETCorePlatformsPackageVersion>2.1.0</BundledNETCorePlatformsPackageVersion>
    <BundledAspNetCoreAllTargetFrameworkVersion>2.1</BundledAspNetCoreAllTargetFrameworkVersion>
    <BundledAspNetCoreAllPackageVersion>2.1.1-rtm-30818</BundledAspNetCoreAllPackageVersion>
    <BundledAspNetCoreAppTargetFrameworkVersion>2.1</BundledAspNetCoreAppTargetFrameworkVersion>
    <BundledAspNetCoreAppPackageVersion>2.1.1-rtm-30818</BundledAspNetCoreAppPackageVersion>
    <NETCoreSdkVersion>2.1.400-preview-009010</NETCoreSdkVersion>
    <_NETCoreSdkIsPreview>true</_NETCoreSdkIsPreview>

    <!-- Default patch versions for each minor version of ASP.NET Core -->
    <DefaultPatchVersionForAspNetCoreAll2_1>2.1.1-rtm-30818</DefaultPatchVersionForAspNetCoreAll2_1>
    <DefaultPatchVersionForAspNetCoreApp2_1>2.1.1-rtm-30818</DefaultPatchVersionForAspNetCoreApp2_1>
    <UseBundledNETCoreAppPackageVersionAsDefaultNetCorePatchVersion>false</UseBundledNETCoreAppPackageVersionAsDefaultNetCorePatchVersion>

    <!-- Latest patch versions for each minor version of .NET Core -->
    <LatestPatchVersionForNetCore1_0 Condition="'$(LatestPatchVersionForNetCore1_0)' == ''">1.0.11</LatestPatchVersionForNetCore1_0>
    <LatestPatchVersionForNetCore1_1 Condition="'$(LatestPatchVersionForNetCore1_1)' == ''">1.1.8</LatestPatchVersionForNetCore1_1>
    <LatestPatchVersionForNetCore2_0 Condition="'$(LatestPatchVersionForNetCore2_0)' == ''">2.0.7</LatestPatchVersionForNetCore2_0>
  </PropertyGroup>
</Project>

```